circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3Copier.java

```diff
@@ -292,7 +293,7 @@ private Copy submitCopyJob(CopyJobRequest copyJob) {
                   copyObjectRequest.getSourceBucketName(),
                   copyObjectRequest.getSourceKey());
           LOG
-              .debug("Copy failed with exception:", e);
+              .info("Copy failed with exception:", e);
```

Comment:
I'm uncertain why this was on debug, maybe because if it fails it tries again? It feels more like it should be on warn but I don't know how often this happens and how noisy it could be? Alternatively put it back the way it was.

Reply:
It tries 3 times by default so could end up printing quite a lot out